# action can be "lock" or "unlock"
string action

# ID of the AGV on which to lock a movable tray
string agv_id

---
bool success
string message